docs/DocGenerator/config.yaml

```diff
@@ -22,24 +20,15 @@ Ignore paths:
   - "../../tests/integration/test_wazuh_db/data"
   - "/data/*"
 
-Valid tags:
-  - DB
-  - Feeds
-  - VulDet
-  - Vulnerability Detector
-
 Output fields:
   Module:
     Mandatory:
       - Brief
-      - Metadata:
-        - Modules
-        - Daemons
-        - Operating System
-        - Tiers
-        - Demo:
-          - One
-          - Two
+      - Metadata
+      - Modules
```

Comment:
Isn't it a little bit confusing to have the `metadata` fields out of it?

Reply:
This change was aimed to simplify the configuration file and make it more flexible. Having a tree of allowed values was more strict to write in the documentation and harder to parse in the code.
We will check if it is required to modify this.

Reply:
Now, the Optional and Mandatory fields have a tree format as suggested.